introduction-combining-data-pandas-merge-join-and-concat/pandas_merge.ipynb

```diff
@@ -0,0 +1,12299 @@
+       "      <td>10</td>\n",
+       "      <td>15</td>\n",
+       "      <td>TWENTYNINE PALMS CA US</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-666</td>\n",
+       "      <td>-66.6</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-666</td>\n",
+       "      <td>-66.6</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-666</td>\n",
+       "      <td>-66.6</td>\n",
+       "      <td>0.05</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.05</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>100</td>\n",
+       "      <td>44</td>\n",
+       "      <td>12</td>\n",
+       "      <td>3</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>0</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>0</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>3</td>\n",
+       "      <td>GHCND:USC00049099</td>\n",
+       "      <td>TWENTYNINE PALMS CA US</td>\n",
+       "      <td>602</td>\n",
+       "      <td>34.12806</td>\n",
+       "      <td>-116.03694</td>\n",
+       "      <td>20100104</td>\n",
+       "      <td>6</td>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>-7777</td>\n",
+       "      <td>-7777</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>-7777</td>\n",
+       "      <td>1</td>\n",
+       "      <td>2</td>\n",
+       "      <td>5</td>\n",
+       "      <td>7</td>\n",
+       "      <td>10</td>\n",
+       "      <td>15</td>\n",
+       "      <td>TWENTYNINE PALMS CA US</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999.0</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999.0</td>\n",
+       "      <td>-6.66</td>\n",
+       "      <td>-9999</td>\n",
+       "      <td>-9999.0</td>\n",
+       "      <td>0.07</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.07</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>101</td>\n",
+       "      <td>45</td>\n",
+       "      <td>12</td>\n",
+       "      <td>3</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>4</td>\n",
+       "      <td>GHCND:USC00049099</td>\n",
+       "      <td>TWENTYNINE PALMS CA US</td>\n",
+       "      <td>602</td>\n",
+       "      <td>34.12806</td>\n",
+       "      <td>-116.03694</td>\n",
+       "      <td>20100105</td>\n",
+       "      <td>6</td>\n",

```

Comment:
For Pandas 0.25.3 this cell raises a `FutureWarning` about sorting the indices that you may want to address. What I see is:

```python
<ipython-input-28-40199f41c038>:1: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=False'.

To retain the current behavior and silence the warning, pass 'sort=True'.

  pd.concat([climate_precip, climate_temp])
```